python/tvm/contrib/hexagon/session.py

```diff
@@ -109,7 +110,7 @@ def device(self):
 
         return self._device
 
-    def upload(self, local_path: Union[str, pathlib.Path], remote_filename: str):
+    def upload(self, local_path: Union[str, pathlib.Path], remote_filename: str) -> pathlib.Path:
```

Comment:
I need this change to make this approach work. It is used in `load_module` function.

Reply:
Is this related to the change from allowing just filenames to requiring a full remote path, so that the caller would have somewhere from which they could learn the full remove path?